From 0019cb76cd430a6e62bc08872c041a043eae8595 Mon Sep 17 00:00:00 2001
From: Jesper Raemaekers <jesper.raemaekers@wefabricate.com>
Date: Sat, 14 Sep 2024 08:32:10 +0200
Subject: [PATCH] API to acceptance

---
 custom_components/weheat/const.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/custom_components/weheat/const.py b/custom_components/weheat/const.py
index fa1b17f..0c1ec01 100644
--- a/custom_components/weheat/const.py
+++ b/custom_components/weheat/const.py
@@ -8,12 +8,12 @@ ENTRY_TITLE = "Weheat cloud"
 ERROR_DESCRIPTION = "error_description"
 
 OAUTH2_AUTHORIZE = (
-    "https://auth.weheat.nl/auth/realms/Weheat/protocol/openid-connect/auth/"
+    "https://auth.early.weheat.nl/auth/realms/Weheat/protocol/openid-connect/auth/"
 )
 OAUTH2_TOKEN = (
-    "https://auth.weheat.nl/auth/realms/Weheat/protocol/openid-connect/token/"
+    "https://auth.early.weheat.nl/auth/realms/Weheat/protocol/openid-connect/token/"
 )
-API_URL = "https://api.weheat.nl"
+API_URL = "https://api.early.weheat.nl"
 OAUTH2_SCOPES = ["openid", "offline_access"]
 
 
-- 
2.43.0

